lifecycle/lifecycle-runtime-testing/bcv/native/current.txt

```diff
@@ -0,0 +1,20 @@
+// Klib ABI Dump
```

Comment:
Do we already have native API validation? Or is it a copy from AOSP?
cc @igordmn

Reply:
it is from the cherry-pick